CareKitStore/CareKitStoreTests/CoreDataSchema/TestCoreDataSchema+Migrations.swift

```diff
@@ -48,14 +48,15 @@ class TestCoreDataSchemaMigrations: XCTestCase {
     ///   Tasks: 3
     ///   Outcomes: 3
     ///   OutcomeValues: 3
-    func testMigrationFrom2_0to2_1() throws {
+    func testMigrationFrom2_0to3_0() throws {
```

Comment:
Removed testMigrationFrom2_0to2_1 because we are not using CareKitStore 2.1. Created new tests for the previous stores to check if their migration was successfull.